configurations/rtd_environment.yml

```diff
@@ -29,7 +29,7 @@ dependencies:
   - hypothesis
   - nbsphinx
   - sphinx_rtd_theme
-  - sphinxcontrib-bibtex
+  - sphinxcontrib-bibtex<2.0.0
```

Comment:
Why is this needed?

Reply:
Seems that a recent update to sphinxcontrib breaks the workflow, and this is a quick fix so that the document builds. I can take a look at it later, if you want.